Frontend/ui-customer/src/Flow.purs

```diff
@@ -1435,8 +1435,12 @@ homeScreenFlow = do
       enterMobileNumberScreenFlow -- Removed choose langauge screen
     SUBMIT_RATING state -> do
       liftFlowBT $ logEventWithMultipleParams logField_ "ny_user_ride_give_feedback" $ [ { key: "Rating", value: unsafeToForeign state.data.ratingViewState.selectedRating } ]
-      void $ lift $ lift $ fork $ HelpersAPI.callApi $ Remote.makeRideFeedBackReq state.data.rideRatingState.rideId state.data.rideRatingState.feedbackList
-      void $ lift $ lift $ fork $ HelpersAPI.callApi $ getfeedbackReq state
+      void $ lift $ lift $ fork do
+        (_ :: (Either ErrorResponse RideFeedbackRes)) <- HelpersAPI.callApi $ Remote.makeRideFeedBackReq state.data.rideRatingState.rideId state.data.rideRatingState.feedbackList
```

Comment:
@RitikaHanish  can't we do void here instead?

Reply:
No, Actually we need to typecast, as in we need to explicitly tell that what response type are we expecting, so that it can decode it into that.

Reply:
ohh got it. 👍🏽